nms/packages/magmalte/app/theme/default.js

```diff
@@ -242,6 +242,11 @@ export default createMuiTheme({
         margin: '0px',
       },
     },
+    MuiAlert: {
+      root: {
+        marginBottom: '15px',
```

Comment:
nit: Typically spacing is done in multiples of pixels divisible by two, often a multiple of 4px. Just convention.
```suggestion
        marginBottom: '16px',
```